index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <!-- this line autocompleted. checked for relevance -->
+    <title>Java Calculator</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+    <!-- <script src='main.js'></script> -->
+</head>
+<body>
+    <header class="calculator-title">
+        <h1>JAVASCRIPT CALCULATOR PROJECT</h1>
+        <h2>Clint Jordan Momentum Team 4</h2>
+    </header>
+    <!-- originally had calc layout in unordered list to make outline. 
+        turns out when changing list items "li" all occurences, text is included.
+        Changed "li" to "buttons" and saw it changed my name "Cbuttont" and my link ref to "buttonk"
+        note for future reference.-->
+<main>
+    <div class="calculator-layout">
+       
+            <button class="calc-buttons letterC">C</class></button>
+            <div id="output" class="display output"></div>
+            <!-- line 25 note: this class will be blank for the output columns 2 to 5 in CSS-->
+            <button class="calc-buttons number7">7</button>
+            <button class="calc-buttons number8">8</button>
+            <button class="calc-buttons number9">9</button>
+            <button class="function-buttons division">/</button>
+            <button class="calc-buttons number4">4</button>
+            <button class="calc-buttons number5">5</button>
+            <button class="calc-buttons number6">6</button>
+            <button class="function-buttons multiplication">*</button>
+       
+
+        
+            <button class="calc-buttons number1">1</button>
+            <button class="calc-buttons number2">2</button>
+            <button class="calc-buttons number3">3</button>
+            <button class="function-buttons subtraction">-</button>
+            <button class="calc-buttons number0">0</button>
+            <button class="function-buttons decimal">.</button>
+            <button class="function-buttons equals">=</button>
+            <button class="function-buttons addition">+</button>
+
+    </div>
+</main>
+<script async src='main.js'></script>
```

Comment:
You need to put this at the bottom of the `body` or add `async`, but both aren't necessary.